### PR TITLE
Pay ID / XpringClient demos

### PR DESCRIPTION
**Pull request by keefertaylor**
## High Level Overview of Change

Implement demos for PayID and XpringClient. 

### Context of Change

Implements additional demos for PayID and XpringClient. 

Note that we're keeping these features out of documentation for the time being, thus no updates to the README, etc. 

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

## Before / After

Moar demos!

## Test Plan

CI

### CODE REVIEW

File: node/src/index-xpring.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+function statusCodeToString(status) {
+    switch (status) {
+      case TransactionStatus.Succeeded:
+        return "SUCCEEDED"
+      case TransactionStatus.Failed: 
+        return "FAILED"
+      case TransactionStatus.Pending:
+        return "PENDING"
+      case TransactionStatus.Unknown:
```
Comment (hbergren) [inline node/src/index-xpring.js]:
```suggestion
      case TransactionStatus.Unknown:
```

File: node/src/index-xpring.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            return "Testnet"
+        case XRPLNetwork.Main:
+            return "Mainnet"
+        default:
+            return "Unknown Network"
+    }
+}
+
+function statusCodeToString(status) {
+    switch (status) {
```
Comment (hbergren) [inline node/src/index-xpring.js]: If you want to be _super clever_, you may be able to rewrite this:
https://www.typescriptlang.org/docs/handbook/enums.html#reverse-mappings

I think that would look like this:
```js
function statusCodeToString(status) {
  return TransactionStatus(status).toUpperCase() || "UNKNOWN"
}
```

Comment (keefertaylor) [reply node/src/index-xpring.js]: "Keep in mind that string enum members do not get a reverse mapping generated at all."

I have no idea why this is and it's a total pain.

Comment (hbergren) [reply node/src/index-xpring.js]: Isn't `TransactionStatus` a numeric enum though?

Comment (hbergren) [reply node/src/index-xpring.js]: https://github.com/xpring-eng/Xpring-JS/blob/87f065ae95b4c05181ca61fa3c888f45fa826205/src/transaction-status.ts

**Comment by keefertaylor** [comment]
Holding this for Xpring-JS  3.0.4

**Pull request merged**